src/thinkspace/api/thinkspace-readiness-assurance/app/models/thinkspace/readiness_assurance/assessment.rb

```diff
@@ -23,6 +27,24 @@ def self.find_by_authables(authables_scope, ra_type)
       # ### Settings.
       # ###
 
+      def sync_assessments
+        ## get my partner
+        ## set my settings n questions n thangs
+        phase      = self.authable
+        assignment = Thinkspace::Casespace::Assignment.find(phase.assignment_id)
+
+        if assignment.sync_rat
+          assessments = Thinkspace::ReadinessAssurance::Assessment.where(authable: assignment.thinkspace_casespace_phases).without(self)
```

Comment:
Yeah, I'm not sure. I don't know if there's a case where we'd be working with multiple RATs per assignment, but that could present some issues.